okhttp/src/jvmTest/java/okhttp3/DispatcherTest.java

```diff
@@ -27,6 +32,24 @@ public final class DispatcherTest {
       .eventListenerFactory(clientTestRule.wrap(listener))
       .build();
 
+  final @Nullable Comparator<Call> priorityOrder = (c1, c2) -> {
+    final String priorityTag = "priority";
+
+    Object t1 = c1.request().tag();
```

Comment:
For this example can we use a typed tag? This form is a single tag per request, better to show with a separate priority tag.

Reply:
@swankjesse as a follow up PR, what do you think about a simple batteries included Priority model with Comparator? Or leave as IKEA flatpack for now?

Reply:
HTTP/2 priority is extremely powerful and in theory we could integrate that all the way to the server. If we support this, we also wanna leave room for that.

Reply:
OK separate tag, that maybe uses this feature.